src/main/scala/samtools.scala

```diff
@@ -4,7 +4,7 @@ import ohnosequences.statika._, bundles._, instructions._
 import java.io.File
 
 
-abstract class Samtools(val version: String)
+abstract class Samtools(val version: String, cdevel : Cdevel)
```

Comment:
if you follow my advice from https://github.com/ohnosequences-bundles/cdevel/pull/1/files#r46300021, this change is not needed :wink: